testing/regress/ecl/redislockingtest.ecl

```diff
@@ -0,0 +1,63 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2014 HPCC Systems.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+IMPORT * FROM lib_redis.locking;
+IMPORT FlushDB FROM lib_redis.sync;
+
+STRING servers := '--SERVER=127.0.0.1:6379';
+
+//SetBoolean(servers, 'b', TRUE);
+//GetBoolean(servers, 'b');
+
+UNSIGNED lock := GetLockObject(servers, 'pi');
+//GetLockObject(servers, 'pi');
+//GetLockObject(servers, 'pi');
+NOFOLD(MissThenLock(lock));
+NOFOLD(MissThenLock(lock));
+
+
+REAL pi := 3.14159265359;
```

Comment:
This test isn't testing the locking in the way that the user would expect to use it.  It needs to match the real-world example which was included in the README.